src/nodes/enqueue.js

```diff
@@ -6,9 +6,12 @@ module.exports = function(RED) {
     RED.nodes.createNode(this, config);
     var node = this;
     node.on('input', function(msg, send, done) {
+      let priorityValue = v_resolve(config.priority, config.priorityType, this.context(), msg);
+
       appendVerb(msg, {
         verb: 'enqueue',
         name:  v_resolve(config.queue, config.queueType, this.context(), msg),
+        priority:  (/^(0|[1-9][0-9]*)$/.test(priorityValue)) ? parseInt(priorityValue) : null,
```

Comment:
`/^\d+$/` should be enough

Reply:
The reason why I chose for this regex was to avoid values starting with 0. Changed it to just digits now.